glean-core/metrics.yaml

```diff
@@ -32,6 +32,7 @@ glean.baseline:
       - baseline
     description: |
       The locale of the application (e.g. "es-ES").
+      If the locale can't be determined on the system, the value is "C".
```

Comment:
Can we add a small comment explaining what's "C" and why it is a meaningful value?

Reply:
Hey @mdboom , this was not updated - was this intentional?

Reply:
Oops.  Should have updated this comment as well.